doctr/transforms/modules/tensorflow.py

```diff
@@ -385,11 +384,14 @@ def extra_repr(self) -> str:
 
     @tf.function
     def __call__(self, img: tf.Tensor) -> tf.Tensor:
-        sigma = random.uniform(self.std[0], self.std[1])
-        return tfa.image.gaussian_filter2d(
-            img,
-            filter_shape=self.kernel_shape,
-            sigma=sigma,
+        return tf.squeeze(
+            _gaussian_filter(
+                img[tf.newaxis, ...],
+                kernel_size=self.kernel_shape,
+                sigma=np.random.uniform(self.std[0], self.std[1]),
```

Comment:
For performance reason, it would be better to keep `random.uniform` if you sample only one number

Reply:
But random.uniform is tracked as possible security issue (codacity log)

Reply:
https://github.com/mindee/doctr/runs/15116301840